Multisig/UI/UI Library/BrowseLinkButton.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  BrowseLinkButton.swift
+//  Multisig
+//
+//  Created by Moaaz on 7/15/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct BrowseLinkButton: View {
```

Comment:
we already have the `LinkButton` and `BrowserLink`, maybe that will work? I'm getting confused by almost identical names :-)

Reply:
I agree, but each one has different style and usage,